tests/TestHelpers/WebDavHelper.php

```diff
@@ -545,6 +549,7 @@ public static function getPersonalSpaceIdForUser(string $baseUrl, string $user,
 	 * @param Client|null $client
 	 * @param array|null $urlParameter to concatenate with path
 	 * @param string|null $doDavRequestAsUser run the DAV as this user, if null its same as $user
+	 * @param bool|null $forNonExistingUser run the DAV as a non-existing user
```

Comment:
the default is false so I think, better to not allow null
```suggestion
	 * @param bool $forNonExistingUser run the DAV as a non-existing user
```

Reply:
true. fixed!